File: conectaBrenannd-api/conectaBrenannd-api/src/main/java/com/conecta/brenannd/service/IComunicadosService.java
package com.conecta.brenannd.service;

import com.conecta.brenannd.entity.Comunicados;

public interface IComunicadosService {

	Comunicados save(final Comunicados entity);
	
	Comunicados update(final long id, final Comunicados entity);
	
	void delete(final long id);
}

File: conectaBrenannd-web/src/app/components/entity/comunicados/service/icomunicados.service.ts
import { Observable } from "rxjs";
import { Comunicados } from "../models/comunicados.models";

export interface IComunicadosService {
  save(request: Comunicados): Observable<Comunicados>;

  update(id: number, request: Comunicados): Observable<Comunicados>;

  delete(id: number): Observable<void>;

  list(): Observable<Comunicados[]>;

  findByID(id: number): Observable<Comunicados>;
}
